cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ParameterUtils.java

```diff
@@ -575,6 +577,52 @@ static Pattern topic(HttpServletRequest request) {
     return parameterString == null ? null : Pattern.compile(request.getParameter(parameterString));
   }
 
+  @SuppressWarnings("unchecked")
+  static Map<Short, Pattern> topicPatternByReplicationFactorFromBody(HttpServletRequest request) {
```

Comment:
Would it be possible to make this function `private` -- i.e. is it expected to be used by a function other than `topicPatternByReplicationFactor()`?